src/upwind_operators.jl

```diff
@@ -169,19 +169,6 @@ and coefficients of Mattsson (2017)
   Diagonal-norm upwind SBP operators.
   Journal of Computational Physics 335, pp. 283-310.
   (upwind coefficients plus)
-
-julia> Matrix(D.central)
```

Comment:
Sorry, there has been a misunderstanding. I would like to keep the output in the docs so that people can see what's going on - but I am open to removing them from the doctests, i.e., the output may look differently in other Julia versions.

Reply:
Doctest + julia block sounds good to me 👍